packages/playwright-core/src/protocol/serializers.ts

```diff
@@ -139,9 +139,9 @@ function innerSerializeValue(value: any, handleSerializer: (value: any) => Handl
     return { bi: value.toString() };
   if (isError(value)) {
     const error = value;
-    if ('captureStackTrace' in globalThis.Error) {
+    if (error.stack?.startsWith(error.name + ': ' + error.message)) {
```

Comment:
If it's v8-specific, can we check for the browser name instead ?

Reply:
remove this?